command/agent/agent.go

```diff
@@ -943,13 +925,10 @@ func (a *Agent) ResumeSync() {
 	a.state.Resume()
 }
 
-// Returns the coordinate of this node in the local pool (assumes coordinates
+// GetLANCoordinate returns the coordinate of this node in the local pool (assumes coordinates
 // are enabled, so check that before calling).
-func (a *Agent) GetCoordinate() (*coordinate.Coordinate, error) {
-	if a.config.Server {
-		return a.server.GetLANCoordinate()
-	}
-	return a.client.GetCoordinate()
+func (a *Agent) GetLANCoordinate() (*coordinate.Coordinate, error) {
```

Comment:
Pls note that the server has `GetLANCoordinate` and `GetWANCoordinate` whereas the client has only `GetCoordinate`. Since the agents' `GetCoordinate` method called `GetLANCoordinate` on the server and clients don't have WAN coordinates I've renamed `GetCoordinate` to `GetLANCoordinate` on both client and agent.